src/cohort.js

```diff
@@ -0,0 +1,65 @@
+class CohortManager {
+  constructor() {
+    this.cohorts = []
+  }
+
+  createCohort(name) {
+    const newCohort = {
+      cohortName: name,
+      students: []
+    }
+    this.cohorts.push(newCohort)
+    return this.cohorts
+  }
+
+  searchCohort(name) {
+    const targCoh = this.cohorts.find((c) => c.cohortName === name)
+    if (!targCoh) {
+      throw new Error('Cohort does not exist')
+    }
+    return targCoh
+  }
+
+  createStudent(coh, firstName, lastName, gitUserName, email) {
+    const tInd = this.cohorts.findIndex((c) => c.cohortName === coh)
+    if (tInd === -1) {
+      throw new Error('Cohort does not exist')
+    }
+    const newId = this.cohorts[tInd].students.length + 1
```

Comment:
What if you remove a student from cohort, does all the id's change because the .length is changed ?